dwitter/static/js/ajax-handling.js

```diff
@@ -64,7 +64,10 @@ var loadComments = function() {
       }
       new_comment_list += getCommentHTML(comment);
     }
-    $comment_section[0].innerHTML = new_comment_list + $comment_section[0].innerHTML;
+    $(comment_section)
+      .html(new_comment_list + comment_section.innerHTML)
+      .promise()
+      .done(Waypoint.refreshAll);
```

Comment:
Since the comment section's HTML is updated asynchronously, we need to wait until the update is finished before calling `Waypoint.refreshAll`. This is accomplished by converting the call to `html` into a promise, then waiting for the promise to resolve.